iosApp/iosApp/Modules/Dashboard/DashboardInteractor.swift

```diff
@@ -205,7 +205,7 @@ extension DefaultDashboardInteractor: DashboardInteractor {
 
 // MARK: - Listeners
 
-extension DefaultDashboardInteractor: NetworksListener, WalletListener, ActionsServiceListener {
+extension DefaultDashboardInteractor: NetworksListener, WalletListener, ActionsListener {
```

Comment:
Prefer `ActionsListener ` to `ActionsServiceListener `

Reply:
IMO here we should keep the name of the `protocol` service as a prefix `<protocol>Listener`, in this instance `ActionsServiceListener` or whatever this is, is more clear that you are registering to the service, and also we could have other listeners that could clash with that name if we follow this naming convention, eg: `ActionsInteractorListener` -> `ActionsListener` too?.